Sources/Multilines/UITextView+Multiline.swift

```diff
@@ -43,6 +43,10 @@ extension UITextView: ContainsMultilineText {
     var numberOfLines: Int {
         -1
     }
+
+    var textAlignment: NSTextAlignment {
```

Comment:
just to know, why do you provide a default value for the text views? 🤔

Reply:
Good catch! I'll remove this 🙈